Classes/Hooks/ItemsProcFunc.php

```diff
@@ -106,12 +106,12 @@ public function user_orderBy(array &$config): void
             $flexformConfig = GeneralUtility::xml2array($row['pi_flexform']);
 
             // check if there is a flexform configuration
-            if (isset($flexformConfig['data']['sDEF']['lDEF']['switchableControllerActions']['vDEF'])) {
-                $selectedActionList = $flexformConfig['data']['sDEF']['lDEF']['switchableControllerActions']['vDEF'] ?? '';
-                // check for selected action
-                if (str_starts_with($selectedActionList, 'Category')) {
+            if (isset($flexformConfig['data']['sDEF']['lDEF'])) {
+                $selectedPlugin = strtolower($row['CType']) ?? '';
+                // check for selected plugin
+                if (str_contains($selectedPlugin, 'category')) {
```

Comment:
we can compare with the relevant plugin keys directly, no need to use a str_contains